proxy/http2/HPACK.cc

```diff
@@ -227,7 +256,20 @@ namespace HpackStaticTable
   {
     HpackLookupResult result;
 
-    for (unsigned int index = 1; index < TS_HPACK_STATIC_TABLE_ENTRY_NUM; ++index) {
+    // Limit the search range of static table
+    unsigned int start = 1;
+    unsigned int end   = TS_HPACK_STATIC_TABLE_ENTRY_NUM;
+    if ('a' <= header.name[0] && header.name[0] <= 'z') {
```

Comment:
Since we're optimizing, does it help to grab the first character once and use that?

```
char const first_char = header.name[0];
```